x-pack/test/functional_with_es_ssl/apps/discover/search_source_alert.ts

```diff
@@ -289,8 +289,8 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
     await testSubjects.click('indexPattern-manage-field');
     await PageObjects.header.waitUntilLoadingHasFinished();
 
-    const titleElem = await testSubjects.find('currentIndexPatternTitle');
-    expect(await titleElem.getVisibleText()).to.equal(dataView);
+    const titleElem = await testSubjects.find('createIndexPatternTitleInput');
```

Comment:
These tests change because its opening the flyout instead of navigating to data view management.